decidim-accountability/app/overrides/models/decidim/category.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Add additional ransacker functionality to category model.
+
+# Categories serve as a taxonomy for components to use for while in the
+# context of a participatory process.
+Decidim::Category.class_eval do
```

Comment:
Would it be useful to add directly to the category model? This way we can reuse this in other admin sections if we want to sort or search by Category.

Reply:
@oriolgual Yes. It would be useful already in the admin : process/assembly/conference : components : proposals index view, for example. We will make this change, and for scope model (below).